src/main/java/org/testng/reporters/jq/Main.java

```diff
@@ -80,7 +83,7 @@ public void generateReport(List<XmlSuite> xmlSuites, List<ISuite> suites,
             if (is == null) {
               throw new AssertionError("Couldn't find resource: " + fileName);
             }
-            Files.copyFile(is, new File(m_outputDirectory, fileName));
+            Files.copyFile(is, new File(m_outputDirectory, stripPrefixfromFileName(fileName)));
```

Comment:
We can actually avoid introducing this new method `stripPrefixfromFileName` if we resorted to something like below

* revert the appending of `TESTNG_RESOURCE_PREFIX` with all the entries in `RESOURCES` array.

* Change : `try (InputStream header = getClass().getResourceAsStream(REPORT_HEADER_FILE))`
To : `try (InputStream header = getClass().getResourceAsStream("/"+TESTNG_RESOURCE_PREFIX + "header")) 

* Change : `try (InputStream is = getClass().getResourceAsStream("/" + fileName)) `
To : `try (InputStream is = getClass().getResourceAsStream("/" + TESTNG_RESOURCE_PREFIX+ fileName)) `

* revert `Files.copyFile(is, new File(m_outputDirectory, stripPrefixfromFileName(fileName)));`
back to `Files.copyFile(is, new File(m_outputDirectory, fileName));`

That way the relative folder need not be appended to any of the entries within `RESOURCES` because its being used only when we are trying to get the resource as a stream.